talent/sourcerers/smola.md

```diff
@@ -0,0 +1,41 @@
+# Santiago Mola
+
+**General**
+- What's your name? How do your friends call you? 
+  - Santiago Moisés, Santi
+- When were you born?
+  - 1987
+- Where are you from? Where have you lived?
+  - [Alicante](https://en.wikipedia.org/wiki/Alicante) → [Valencia](https://en.wikipedia.org/wiki/Valencia) → [Madrid](https://en.wikipedia.org/wiki/Madrid) ([Spain](https://en.wikipedia.org/wiki/Spain))
+- What languages (natural and programming ones) do you know?
+  - Spanish, English, Catalonian, Go, Java, Scala, Python and a few others I am not proficient with or I would prefer to forget.
+- Would like to share your Twitter, Instagram or Medium or any other social media account?
+  - [@mola\_io](https://twitter.com/mola_io)
+
+**Work**
+- What are you going to be doing at source{d}?
+  - [Architecting](http://devopsreactions.tumblr.com/post/46582414073/the-senior-architect-is-at-the-whiteboard).
+- Why did you join source{d}?
+  - [Awesome team](https://giphy.com/gifs/bear-power-team-l3q2Wl7Wpz09Z5hfi) working on [crazy things](https://www.youtube.com/watch?v=hKoB0MHVBvM).
+- Where will you work from?
+  - Madrid, office and remote.
+- If you could only save one programming language, which one would it be? 
+  - I would save C++, so implementations for most other languages would still exist. Then I would code in something other than C++.
+- If you wanted to do an investment, what company would you choose?
+  - You cannot invest in a State Planning Agency.
+
+**Other**
+- What are your hobbies and what do you like about them?
+  - Most computer-related stuff.
+- What was the last lie you told?
+  - Something in this questionnaire.
+- What was your first thought when you woke up this morning?
+  - $@!# what happened to the alarm?
+- What question do you hate to answer?
+  - *Tell me three of your virtues and three defects.*
+- What’s your favorite knock-knock joke?
+  - Not a common thing in Spain as far as I know.
```

Comment:
Maybe we should change it to: what's your favorite "camarero!" song?

Reply:
Or maybe something that has a lower cultural bias.

Reply:
It seems that no spaniard could answer the question? On the other hand, the way @gomesfernanda [adapted the answer](https://github.com/src-d/guide/blob/master/talent/sourcerers/gomesfernanda.md] was hilarious. Although I guess it works only in Portuguese and Spanish.

Reply:
> Hi, I'm Tunisian, and I'm here to kill you!
> To what?
> Tunisian